website/server/controllers/api-v3/pushNotifications.js

```diff
@@ -0,0 +1,99 @@
+import { authWithHeaders } from '../../middlewares/api-v3/auth';
+import _ from 'lodash';
+import {
+  BadRequest,
+  NotAuthorized,
+} from '../../libs/api-v3/errors';
+
+let api = {};
+
+/**
+ * @apiIgnore
+ * @api {post} /api/v3/user/add-push-device Add a push device to a user
```

Comment:
I think this could just be `/user/push-device` the POST implies that you are adding one.